### PR TITLE
Issue 112: Use preview_url field from Narration model to render book details page

### PR DESCRIPTION
**Pull request by frombrest**
#112  Use preview_url field from Narration model to render book details page

image

### CODE REVIEW

File: templates/books/book-detail.html [hunk 1]
```diff
@@ -84,11 +84,6 @@ <h1 class="h2">{% dtranslate book.title %}</h1>
                 {% endif %}
                 {% cite_source book.description_source "cit-description" %}
             </div>
-
-            {% if book.preview_url %}
-                {% translate "Першую частку можна паслухаць бясплатна на" %} <a href="{{ book.preview_url }}" target="_blank">YouTube</a>.
-            {% endif %}
```
Comment (nbeloglazov) [inline templates/books/book-detail.html:90]: I need to make sure I ran the migration and fix the cases with 2+ narrations before deploying this, right?

**Review by nbeloglazov — APPROVED** [review]

File: templates/books/book-detail.html (hunk 1 above)
Comment (frombrest) [reply templates/books/book-detail.html:90]: of course

Comment (nbeloglazov) [reply templates/books/book-detail.html:90]: Sounds good.

**Comment by nbeloglazov** [comment]
A test would be great to have.

**Comment by frombrest** [comment]
> A test would be great to have.

sure, that is on me

**Pull request merged**